JavaScript/Topics/15.Events/README.md

```diff
@@ -0,0 +1,156 @@
+| mousedown       | onmousedown   | When the mouse button is pressed over the element   |
+| mouseup         | onmouseup     | When the mouse button is released over the element  |
+| mousemove       | onmousemove   | When the mouse movement takes place.                |
+
+# Keyboard Events
+
+| Event Performed | Event Handler       | Description                                 |
+|-----------------|---------------------|---------------------------------------------|
+| Keydown & Keyup | onkeydown & onkeyup	| When the user press and then release the key|
+
+# onclick Event
+The onclick event occurs when the user clicks on an element.
+
+This is the most frequently used event type which occurs when a user clicks the left button of his mouse. You can put your validation, warning etc., against this event type.
+
+### Syntax:
+```js
+// In JavaScript:
+object.onclick = function(){myScript};
+// In JavaScript, using the addEventListener() method:
+object.addEventListener("click", myScript);
+```
+Example:
+
+```js
+// Click on a <p> element to change its text color to blue:
+<p id="demo" onclick="myFunction()">Click me to change my text color.</p>
+
+<script>
+function myFunction() {
+  document.getElementById("demo").style.color = "blue";
+}
+</script>
+```
+# onsubmit Event
+The onsubmit event occurs when a form is submitted.
+
+You can put your form validation against this event type.
+### Syntax:
+```js
+// In JavaScript:
+object.onsubmit = function(){myScript};
+// In JavaScript, using the addEventListener() method:
+object.addEventListener("submit", myScript);
+```
+
+Example:
+
+```js
+// Execute a JavaScript when a form is submitted:
+<form onsubmit="myFunction()">
+  Enter name: <input type="text">
+  <input type="submit">
+</form>
+```
+<br>
+
+# HTML 5 Standard Events
+
+| Attribute          | Value  | Description                                                                                                  |
+|--------------------|--------|--------------------------------------------------------------------------------------------------------------|
+| Offline            | script | Triggers when the document goes offline                                                                      |
+| Onabort            | script | Triggers on an abort event                                                                                   |
+| onafterprint       | script | Triggers after the document is printed                                                                       |
+| onbeforeonload     | script | Triggers before the document loads                                                                           |
+| onbeforeprint      | script | Triggers before the document is printed                                                                      |
+| onblur             | script | Triggers when the window loses focus                                                                         |
```

Comment:
Link all of these attributes to their W3Schools demo page.
For Example
`[onblur](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onblur)`